tailwind.config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     fontSize: {
       xs: ['0.625rem', 1.6],
       sm: ['0.75rem', 1.5],
-      md: ['0.875rem', 1.4],
+      md: ['0.875rem', '1.25rem'],
```

Comment:
This looks to better match the design, we looked in as many places as we could find it was being used and seems to not cause any alignment issues.